suds/wsse.py

```diff
@@ -24,10 +24,11 @@
 from datetime import datetime, timedelta
 
 try:
-    from hashlib import md5
+    from hashlib import sha256
 except ImportError:
     # Python 2.4 compatibility
-    from md5 import md5
+    from md5 import md5 as sha56
```

Comment:
Can we import md5 as **sha256**, instead of she56 to keep the naming convention consistent.?